guides/workspaces.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Workspace organization"
+description: Learn how to organize your Coder workspaces.
+---
+
+This article describes considerations for deciding how to set up your Coder
+workspaces.
+
+In general, the fewer workspaces per developer, the easier it is for the
+individual developer to manage. However, the complexity of the underlying images
```

Comment:
Maybe add a sentence/link like: "For teams that do not have a complex development workflow, we recommend [one workspace per developer], as it is easiest to adopt."